src/5e-SRD-Levels.json

```diff
@@ -8158,5 +8154,974 @@
+      }
+    ],
+    "spellcasting": {},
+    "subclass_specific": {},
+    "class": {
+      "name": "Cleric",
+      "url": "/api/classes/cleric"
+    },
+    "subclass": {
+      "name": "Life",
+      "url": "/api/subclasses/life"
+    },
+    "url": "/api/subclasses/life/levels/1",
+    "index": 248
+  },
+  {
+    "level": 2,
+    "feature_choices": [],
+    "features": [
+      {
+        "name": "Channel Divinity: Preserve Life",
+        "url": "/api/features/channel-divinity-preserve-life"
+      }
+    ],
+    "spellcasting": {},
+    "subclass_specific": {},
+    "class": {
+      "name": "Cleric",
+      "url": "/api/classes/cleric"
+    },
+    "subclass": {
+      "name": "Life",
+      "url": "/api/subclasses/life"
+    },
+    "url": "/api/subclasses/life/levels/2",
+    "index": 249
+  },
+  {
+    "level": 6,
+    "feature_choices": [],
+    "features": [
+      {
+        "name": "Blessed Healer",
+        "url": "/api/features/blessed-healer"
+      }
+    ],
+    "spellcasting": {},
+    "subclass_specific": {},
+    "class": {
+      "name": "Cleric",
+      "url": "/api/classes/cleric"
+    },
+    "subclass": {
+      "name": "Life",
+      "url": "/api/subclasses/life"
+    },
+    "url": "/api/subclasses/life/levels/6",
+    "index": 250
+  },
+  {
+    "level": 8,
+    "feature_choices": [],
+    "features": [
+      {
+        "name": "Divine Strike",
+        "url": "/api/features/divine-strike"
+      } 
+    ],
+    "spellcasting": {},
+    "subclass_specific": {},
+    "class": {
+      "name": "Cleric",
+      "url": "/api/classes/cleric"
+    },
+    "subclass": {
+      "name": "Life",
+      "url": "/api/subclasses/life"
+    },
+    "url": "/api/subclasses/life/levels/8",
+    "index": 251
+  },
+  {
+    "level": 17,
+    "feature_choices": [],
+    "features": [
+      {
+        "name": "Supreme Healing",
+        "url": "/api/features/supreme-healing"
+      } 
+    ],
+    "spellcasting": {},
+    "subclass_specific": {},
+    "class": {
+      "name": "Cleric",
+      "url": "/api/classes/cleric"
+    },
+    "subclass": {
+      "name": "Life",
+      "url": "/api/subclasses/life"
+    },
+    "url": "/api/subclasses/life/levels/17",
+    "index": 252
+  },
+  {
+    "level": 2,
+    "feature_choices": [],
+    "features": [
+      {
+        "name": "Bonus Cantrip",
+        "url": "/api/features/bonus-cantrip"
+      },
+      {
+        "name": "Natural Recovery",
+        "url": "/api/features/natural-recovery"
+      }
+    ],
+    "spellcasting": {},
+    "subclass_specific": {},
+    "class": {
+      "name": "Druid",
+      "url": "/api/classes/druid"
+    },
+    "subclass": {
+      "name": "Land",
+      "url": "/api/subclasses/land"
+    },
+    "url": "/api/subclasses/land/levels/2",
+    "index": 253
+  },
+  {
+    "level": 6,
+    "feature_choices": [],
+    "features": [
+      {
+        "name": "Land's Stride",
+        "url": "/api/features/lands-stride"
```

Comment:
This URL doesn't exist. I believe you want `/api/features/druid-lands-stride`

Reply:
You may want to do a once over and make sure all of your feature urls exist. I wish the CI covered this check but it doesn't yet.

Reply:
I thought I had checked all of them! I'll fix this one and double check.

Reply:
Done! I've written a script to resolve all the reference URLs and they're all coming back positive :)